docs/.gitignore

```diff
@@ -1,7 +1,7 @@
 _site
 
 # You can clone your "hubble-data" repository to this location for debugging
-hubble-data
+demo-data
```

Comment:
Are you sure this is correct? Demo data should be under version control for the demo website!

Reply:
You’re right. `demo-data` should be removed from the `.gitignore` instead.

Reply:
For the record, I misunderstood the purpose of this line. It’s not for excluding the demo data directory (of course it isn’t, my mistake) but for excluding another local data directory for testing (I should have read the comment more carefully).